### DISCUSSION TITLE
warning: mixed firewall rules in fresh build image

### DISCUSSION THREAD

**Discussion by yyshell** [immortalwrt/immortalwrt#1297] [General]
I build a x86-64 image. when I click firewall status, there is a warning I have mixed firewall rules with iptables and nftables. 
<img width="1165" alt="firewall-rules" src="https://github.com/immortalwrt/immortalwrt/assets/46114966/eccec75e-2e38-4af1-84fc-f03e9db16dc7">
here's my buidinfo config file.:
[config.buildinfo.gz](https://github.com/immortalwrt/immortalwrt/files/15328050/config.buildinfo.gz)
how should I deal with it?

**Comment by yyshell** [comment]
I know why there are iptables rules in my Immoralwrt. ShadowsocksRplus and passwall will add the rules after I run them.

**Comment by fuliaojiyue** [comment]
我编译的也出现这种警告，我也不知道这种警告对使用有没有影响，现在网上搜了一圈，没找到解决的方法。

**Reply by xlighting2017** [reply]
```
# CONFIG_PACKAGE_luci-app-passwall_Iptables_Transparent_Proxy is not set
CONFIG_PACKAGE_luci-app-passwall_Nftables_Transparent_Proxy=y
```
